.github/workflows/main.yml

```diff
@@ -0,0 +1,86 @@
+name: Application Tests
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+env:
+  BUNDLE_PATH: /usr/local/bundle
+  BUNDLE_JOBS: 3
+  BUNDLE_RETRY: 3
+jobs:
+  build:
+    name: Test Application
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        rubygems: [2.6.10, latest]
+    services:
```

Comment:
Wouldn't be easier to just start `docker-compose up -d db cache search` and keep dependencies maintained in `docker-compose.yml` only?